src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/http-foundation": "~2.7|~3.0",
         "symfony/http-kernel": "~2.7|~3.0",
         "symfony/filesystem": "~2.7|~3.0",
-        "symfony/routing": "~2.7|~3.0",
+        "symfony/routing": "~3.0",
```

Comment:
It's only required for tests not the actual functionality (because _scheme/_method would not hurt). Is it possible to have the same dependecy with different versions for `require` and `require-dev`?
I.e. `~2.7|~3.0` in `require` but `~3.0` in `require-dev`?